prestoadmin/package.py

```diff
@@ -90,39 +90,72 @@ def deploy(local_path=None):
 
 
 def _rpm_install(package_path):
-    nodeps = ''
-    if env.nodeps:
-        nodeps = '--nodeps '
+    nodeps = _nodeps_rpm_option()
 
     if 'java8_home' not in env or env.java8_home is None:
-        ret = sudo('rpm -i %s%s' % (nodeps, package_path))
+        sudo('rpm -i %s%s' % (nodeps, package_path))
     else:
         with shell_env(JAVA8_HOME='%s' % env.java8_home):
-            ret = sudo('rpm -i %s%s' % (nodeps, package_path))
+            sudo('rpm -i %s%s' % (nodeps, package_path))
+
 
-    return ret
+def _nodeps_rpm_option():
+    nodeps = ''
+    if env.nodeps:
+        nodeps = '--nodeps '
+    return nodeps
 
 
 def rpm_install(rpm_name):
     _LOGGER.info("Installing the rpm")
-    package_path = os.path.join(constants.REMOTE_PACKAGES_PATH, rpm_name)
-    ret = _rpm_install(package_path)
-    if ret.succeeded:
-        print("Package installed successfully on: " + env.host)
+    _rpm_install(_rpm_path(rpm_name))
+    print("Package installed successfully on: " + env.host)
```

Comment:
Well, not always. If _rpm_install fails the exception should be raised, unless it is called `with quite()` then this will be printed.

Or I don't understand something. 

Anyway, I will revert this check.